index.html

```diff
@@ -0,0 +1,150 @@
+                            <input id="definately" type="radio" value="definately" name="recommend">
+                            Definately
+                        </label> 
+                    </div>
+                    <div>
+                        <label for="maybe">
+                            <input id="maybe"  type="radio" value="maybe" name="recommend">
+                            Maybe
+                        </label>
+                    </div>
+                    <div>
+                        <label for="not_sure">
+                            <input id="not_sure" type="radio" value="not_sure" name="recommend">
+                            Not sure
+                        </label>
+                    </div>
+                </fieldset>
+                    
+            </div>
+            <div class="formElment">
+                <label>What is your favorite feature of freeCodeCamp?</label>
+                <select>
+                    <option disabledbyvalue>Select an option</option>
+                    <option value="Student">Challenges</option>
+                    <option value="job">Projects</option>
+                    <option value="learner">Community</option>
+                    <option value="nothing">Open source</option>
+                </select>
+            </div>
+
+            <div class="formElement">
+                <label for="">What would you like to see improved? </label>
+                <div>
+                    <input type="checkbox" name="" id="">
+                    Front-end Projects
+                </div>
+                <div>
+                    <input type="checkbox" name="" id="">
+                    Back-end Projects
+                </div>
+                <div>
+                    <input type="checkbox" name="" id="">
+                    Data Visualization
+                </div>
+                <div>
+                    <input type="checkbox" name="" id="">
+                    Challenges
+                </div>
+                <div>
+                    <input type="checkbox" name="" id="">
+                    Open Source Community
+                </div>
+                <div>
+                    <input type="checkbox" name="" id="">
+                    Gitter help rooms
+                </div>
+                <div>
+                    <input type="checkbox" name="" id="">
+                    Videos
+                </div>
+                <div>
+                    <input type="checkbox" name="" id="">
+                    City Meetups
+                </div>
+                <div>
+                    <input type="checkbox" name="" id="">
+                    Wiki
+                </div>
+                <div>
+                    <input type="checkbox" name="" id="">
+                    Forum
+                </div>
+                <div>
+                    <input type="checkbox" name="" id="">
+                    Additional Courses
+                </div>
```

Comment:
Please use a < ul > with < li >. It is a great way to organize your site's navigation and the semantic markup will improve accessibility for users that require screen-readers.